client/src/components/Banner.jsx

```diff
@@ -32,34 +32,12 @@ function Banner() {
 
   return (
     <div className="container mx-auto flex flex-col sm:flex-col md:flex-row justify-between py-6 sm:h-28">
-      <div className="sm:hidden">
-        <div className="flex justify-center mb-4">
+      <div className="hidden sm:flex w-full md:w-3/4 justify-start sm:items-end">
+        <div className="flex items-center">
           <Link to="/" className="focus:outline-none">
             <img src={Logo} alt="Recipe King" className="logo" />
           </Link>
         </div>
-        <div className="flex justify-center">
-          <input
-            type="text"
-            placeholder="Search..."
-            value={searchText}
-            onChange={(e) => setSearchText(e.target.value)}
-            onKeyPress={handleKeyPress}
-            className="w-96 shadow appearance-none border rounded py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline mx-1"
-          />
-          <button
-            type="button"
-            onClick={handleSearch}
-            className="bg-blue-500 text-white px-4 py-2 rounded focus:outline-none"
-          >
-            Search
-          </button>
-        </div>
-      </div>
-      <div className="hidden sm:flex w-full md:w-3/4 justify-start sm:items-end">
-        <div className="flex items-center">
-          <img src={Logo} alt="Recipe King" className="logo" />

```

Comment:
itt nem volt megadva a Link-tag meg a to és ezért nem működött a logora kattintás